WcaOnRails/app/models/event.rb

```diff
@@ -54,4 +54,12 @@ def fewest_moves?
   def multiple_blindfolded?
     self.id == "333mbf" || self.id == "333mbo"
   end
+
+  def serializable_hash(options = nil)
```

Comment:
This whole PR is a bit artificial, since it doesn't do anything useful. Makes it hard to review anything other than small parts.

It would be better to keep loose ends out of it, yes.